### DISCUSSION TITLE
Is it possible to ignore specific file for files matching a pattern?

### DISCUSSION THREAD

**Discussion by lukashlobil** [eslint/eslint#16598] [Q&A] [answered]
Couple of my files (in multiple folders) have multiple classes. I want these excluded from my linting rule below. 
```
   "max-classes-per-file": [
          "error",
          {
            "ignoreExpressions": true,
            "max": 1
          }
        ],
```

something like 
`         "ignoreFiles": [*.filetoIgnore.*]`

Is it possible? (yes i know I can type eslint-disable in those files, looking for global solutions. I want other rules to still apply for them.

**Comment by mdjermanovic** [comment, marked as answer]
You can use [Configuration Based on Glob Patterns](https://eslint.org/docs/latest/user-guide/configuring/configuration-files#configuration-based-on-glob-patterns) in your config file to disable the rule for specified files.

```json
"overrides": [
    {
        "files": ["path/to/file.js", "**/*.filetoIgnore.*"],
        "rules": {
            "max-classes-per-file": "off"
        }
    }
]
```

**Reply by adammfrank** [reply]
I don't think this is correct, or it may have changed since the answer was posted. It should be `0` not "off".

**Reply by ghiscoding** [reply]
`off` and `0` are equivalent
